django_dramatiq/management/commands/rundramatiq.py

```diff
@@ -168,12 +184,8 @@ def _get_submodules(self, package):
         package_path = package.__path__
         prefix = package.__name__ + "."
 
-        for _, module_name, is_pkg in pkgutil.walk_packages(package_path, prefix):
-            if is_pkg:
```

Comment:
Why is this branch no longer needed?

Reply:
This was causing duplicate tasks to be registered once you got to sub module of sub module, as `is_pkg` looks like it actually refers to if the file / module is a `__init__.py` file (https://github.com/python/cpython/blob/3.9/Lib/pkgutil.py#L175), and `walk_packages` already walks the tree recursively.

Reply:
Got it, thanks! I see now that you mentioned this in your PR message. Sorry I missed it!